kolibri_explore_plugin/assets/src/customApps.js

```diff
@@ -9,3 +11,37 @@ export const CustomChannelApps = {
 export function getAppNameByID(id) {
   return CustomChannelApps[id] || 'default';
 }
+
+// This should match a file existing on static/thumbnails/NAME.jpg
+export const ThumbApps = [
+  'music',
+  'ted-ed-lessons',
+  'common-sense-student-resources',
+  'carreer-girls',
+  'cspathshala',
+  'touchable-earth',
+  'oceanx',
+  'phet-interactive-simulations',
+  'khan-academy',
+  'open-stax',
+  'pbs-kids',
+  'healthy-living',
+];
+
+export function getBigThumbnail(channel) {
+  if (!channel.title) {
+    return null;
+  }
+
+  const normalName = channel.title
+    .toLowerCase()
+    .replaceAll(/\(.*\)/g, '')
+    .trim()
+    .replaceAll(' ', '-');
```

Comment:
We have a similar logic in `getSlug(title)` in `packages/template-ui/src/utils.js` that can be reused, moving it to the eos-components `utils.js`. It can be done next.